### PR TITLE
Add post, like and comment create actions with respective views

### PR DESCRIPTION
**Pull request by UbdaNam**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Query first user
- [x] Create post actions, views and routes
- [x] Create comment actions, views and routes
- [x] Allow users to like posts
- [x] Setup rubocop and stylelint linters

## Description of the tasks completed 👇 

- [x] Add post, like and comment create actions with respective views
- [x] Install rails gem
- [x] Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯 
  
## How should it be tested

Clone the repository
```
git clone https://github.com/UbdaNam/Blogiverse.git
```
Navigate to the folder
```
cd Blogiverse
```
Checkout the branch
```
git checkout feature_add_form
```
Run migrations
```
rails db:migrate
```
Run server
```
rails s
```

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -12,4 +12,27 @@ def show
...
+    @post.author_id = current_user.id
+    @post.likes_counter = 0
+    @post.comment_counter = 0
+
+    if @post.save
+      redirect_to users_path
+    else
+      render :new
+    end
+  end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/posts_controller.rb]: - [x] I can not create a post, first the URL is wrong, also the post somehow is not creating even when I change the URL to take a user id of 1, kindly watch this [video](https://www.loom.com/share/0dbc0192067c443cb4aca1e23d032373?sid=3b89c7b4-2313-40d7-a025-b29b5a69ca0e) to understand what I am saying

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @UbdaNam  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (UbdaNam) [reply app/controllers/posts_controller.rb]: I Fixed the redirect issue thanks @VICTOR-CHUKWU

Comment (Munsa1) [reply app/controllers/posts_controller.rb]: - [x] Good job implementing the requested change by the previous reviewer, however creating a post is still not possible as there is a typo `undefined method `comment_counter=' causing an error, kindly fix it to `comments_counter`.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @UbdaNam, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by UbdaNam** [comment]
Hi @Munsa1 Thank you so much for reviewing my PR 👍 . I have fixed the issue and submitted re-review please kindly check it.

**Review by sumairq — APPROVED** [review]
### Hi @UbdaNam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Changes implemented properly
✔️Linters are passing
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by UbdaNam** [comment]
Thank you @sumairq so much for reviewing my PR ❤️

**Pull request merged**